src/backend/utils/storage.js

```diff
@@ -47,6 +51,13 @@ module.exports = {
 
   getFeedsCount: () => redis.scard(feedsKey),
 
+  addInvalidFeed: (feed, reason) => {
```

Comment:
The rest of our Redis feed code accepts an `id` vs. a `feed`.  Let's do that here too.